src/backend/user-server/src/main/java/userserver/domain/Status.java

```diff
@@ -0,0 +1,8 @@
+package userserver.domain;
+
+public enum Status {
+
+    ENABLE, // 활성화(이메일 인증 완료)
+    DISABLE, // 비활성화(블락 처리)
+    QUIT // 탈퇴
```

Comment:
혹시 이메일 인증 전에는 어떤 상태일까요?

Reply:
조금 더 활성화, 비활성화, 탈퇴에 맞는 용어를 쓰면 더 좋을 것 같아요!
`ENABLE`, `DISABLE` 은 어떤 기능을 키고 끈다는 느낌이 강하고,
`QUIT` 은 탈퇴 느낌 보다는 종료했다 라는 느낌이 강해보였어요.

```suggestion
    INACTIVE
    ACTIVE,
    BLOCK,
    DELETE
```

Reply:
현재 회원가입 중간에 이메일 인증을 넣어서, 이메일 인증이 되지 않으면 회원가입이 진행되지 않는 로직입니다. 이메일 인증 전에는 어떤 상태인지 정보가 필요할까요?! 몰라서 물어봅니다!

Reply:
@happysoy 아하, 고러면은 `INACTIVE` 는 굳이 필요 없을 것 같네요.